Tests/SwiftParserTest/translated/IfconfigExprTests.swift

```diff
@@ -455,6 +462,19 @@ final class IfconfigExprTests: XCTestCase {
     )
   }
 
+  func testCompileConditionFuncCall() {
+    assertParse(
+      """
+        arch()
+        canImport(a, b, c)
+        os(bogus)
+        targetEnvironment(foo, bar)
+        compiler(a)
+        swift(foo)
+      """
+    )
```

Comment:
Could you create separate `asserParse` calls for all of these? If they fail, I find it’s a lot easier to tell what’s going wrong if the code inside `assertParse` is as minimal as possible.